pulsar-client/src/test/java/org/apache/pulsar/client/impl/conf/ClientConfigurationDataTest.java

```diff
@@ -18,38 +18,109 @@
  */
 package org.apache.pulsar.client.impl.conf;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.ObjectWriter;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.pulsar.client.api.PulsarClientException;
+import org.apache.pulsar.client.impl.auth.AuthenticationBasic;
+import org.apache.pulsar.client.impl.auth.AuthenticationDisabled;
 import org.apache.pulsar.client.impl.auth.AuthenticationToken;
-import org.testng.Assert;
 import org.testng.annotations.Test;
 
 /**
  * Unit test {@link ClientConfigurationData}.
  */
 public class ClientConfigurationDataTest {
 
-    private final ObjectWriter w;
```

Comment:
I know this change was a bit out of scope, but it was the only test in the class and it stuck out a bit (inconsistent) once I added the other test methods. I simplified it for consistency, thats all. Also, the `assertThat::doesNotContain` is perfect for this verification.